cache/disk/disk.go

```diff
@@ -649,6 +649,9 @@ func cacheFilePath(kind cache.EntryKind, cacheDir string, hash string) string {
 // value from the CAS if it and all its dependencies are also available. If
 // not, nil values are returned. If something unexpected went wrong, return
 // an error.
+// TODO Consider separating implementation of cache.AcStore interface, to open up
+//      possibilities combining that functionality with proxies, and also allow
+//      bazel-remote configurations with proxy but no local disk storage?
```

Comment:
@mostynb Does it make sense to bring back the `cache.Cache` interface that the proxies and disk cache used to implement for something like that?

Reply:
@mostynb Is it something more than implementation of GetValidatedActionResult, that you expect will be needed from disk.io, in a proxy-only-mode?

Reply:
No. I’m only thinking that having a separate AcStore interface might be more generic and flexible. Especially if GetValidatedActionResult method would be extracted from disk.go in the future. Or if there would be decorators wrapping only AcStore. But I don’t have much experience of embedding interfaces and structs in golang. It might work with only BlobStore and BlobAcStore interface, if you think it is preferable to avoid AcStore?